coderd/database/migrations/000030_template_version_created_by.up.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+ALTER TABLE ONLY template_versions ADD COLUMN IF NOT EXISTS created_by uuid REFERENCES users (id) ON DELETE RESTRICT;
+
+UPDATE
+    template_versions
+SET
+    template_versions.created_by = (
+        SELECT templates.created_by FROM templates
+        WHERE template_versions.template_id = templates.id
+        LIMIT 1
+    )
+WHERE
+    template_versions.created_by IS NULL;
+
+ALTER TABLE ONLY template_versions ALTER COLUMN created_by SET NOT NULL;
+
+COMMIT;
```

Comment:
@dwahler I am hoping wrapping it in `BEGIN`-`COMMIT` would help to fix the problem caused by the migration. Not really sure if this broke the server though.

Reply:
Interestingly, `make gen` is failing with:
```
panic: reached retry deadline
goroutine 1 [running]:
main.main()
	/home/runner/work/coder/coder/coderd/database/dump/main.go:19 +0x80c
exit status 2
make: *** [Makefile:60: coderd/database/dump.sql] Error 1
```

Reply:
Fixed the bug, the scoping logic was not good.